### PR TITLE
Add Ghc-8.2 in travis

### PR DESCRIPTION
**Pull request by vincenthz**
For now use rc1

### CODE REVIEW

File: .travis/stack-ghc-8.0.yaml [hunk 1]
```diff
@@ -1,66 +1,6 @@
-# This file was automatically generated by 'stack init'
```
Comment (NicolasDP) [inline .travis/stack-ghc-8.0.yaml]: much better indeed

**Comment by NicolasDP** [comment]
Now it's a shame that all the links on haskell.org's download page are dead.

**Comment by NicolasDP** [comment]
It seems the unavailability of the downloads.haskell.org was only temporary https://mail.haskell.org/pipermail/ghc-devs/2017-April/014124.html

**Comment by vincenthz** [comment]
Probably need to be tweak to actually work, but since it's marked as allowed_failure, merging to move on and can revisit this soon

**Pull request merged**